docs/pages/versions/unversioned/sdk/location.mdx

```diff
@@ -141,7 +141,17 @@ To use Geofencing methods, the following requirements apply:
 
 - Location permissions must be granted. On iOS it must be granted with `Always` option.
 - The Geofencing task must be defined in the top-level scope, using [TaskManager.defineTask](task-manager.mdx#taskmanagerdefinetasktaskname-taskexecutor).
-- On iOS, there is a [limit of 20](https://developer.apple.com/documentation/corelocation/monitoring_the_user_s_proximity_to_geographic_regions) `regions` that can be simultaneously monitored.
+
+When using Geofencing, the following platform differences apply:
+
+- <PlatformTag platform="ios" /> Expo Location will report the initial state of the registered
+  geofence(s) at app startup.
+- <PlatformTag platform="ios" /> There is a [limit of
+  20](https://developer.apple.com/documentation/corelocation/monitoring_the_user_s_proximity_to_geographic_regions)
+  `regions` that can be simultaneously monitored.
+- <PlatformTag platform="android" /> You are allowed [up to
+  100](https://developer.android.com/develop/sensors-and-location/location/geofencing) active
+  geofences per app.
```

Comment:
```suggestion
When using Geofencing, the following platform differences apply:

- <PlatformTag platform="android" /> You are allowed [up to
  100](https://developer.android.com/develop/sensors-and-location/location/geofencing) active
  geofences per app.
- <PlatformTag platform="ios" /> Expo Location will report the initial state of the registered
  geofence(s) at app startup.
- <PlatformTag platform="ios" /> There is a [limit of
  20](https://developer.apple.com/documentation/corelocation/monitoring_the_user_s_proximity_to_geographic_regions)
  `regions` that can be simultaneously monitored.
```
Let's document Android info before iOS as per our writing style guide's [preferred platform order](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms)

Reply:
Thanks, @amandeepmittal - I read the guide but missed that one :)

Reply:
No problem :)